com.google.Chrome.metainfo.xml

```diff
@@ -1,223 +1,39 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2017 the Chromium authors, with modifications -->
 <component type="desktop">
-  <id>com.google.Chrome</id>
-  <launchable type="desktop-id">com.google.Chrome.desktop</launchable>
-  <name>Google Chrome</name>
-  <developer_name>Google</developer_name>
-  <summary>The web browser from Google</summary>
-  <metadata_license>CC0-1.0</metadata_license>
-  <project_license>LicenseRef-proprietary</project_license>
-  <url type="homepage">https://chrome.google.com/</url>
-  <description>
-    <p>
-    Google Chrome is a browser that combines a minimal design with sophisticated technology to make the web faster,
-    safer, and easier.
-   </p>
-   <p>NOTE: This wrapper is not verified by, affiliated with, or supported by Google.</p>
-  </description>
-  <screenshots>
-    <screenshot type="default">
-      <image>https://www.gstatic.com/chrome/appstream/chrome-2.png</image>
-    </screenshot>
-  </screenshots>
-  <releases>
-    <release version="129.0.6668.58-1" date="2024-09-16">
-      <description></description>
-    </release>
```

Comment:
These are automatically added by x-checker, not sure if removing them is worth it.